text/0000-lts-release-strategy.md

```diff
@@ -0,0 +1,90 @@
+
+## Release cadence
+
+Hyperledger Fabric will continue the pattern of quarterly minor releases, e.g. v2.0, v2.1, v2.2, etc.
+
+## LTS release cadence
+
+Because a new major release typically has large new features that may not yet be tried by the user community, and because SDKs may lag in support of the new release, it is not expected that a new major release will immediately be designated as a LTS release, for example, v2.0 is not a LTS release.
+
+Each major release will eventually have at least one minor release designated by the Fabric maintainers as the LTS release, e.g. v1.4 for v1.x, v2.2 for v2.x (all references to future versions are hypothetical).
+
+After a LTS release is designated, quarterly releases will continue as normal. The Fabric maintainers will decide whether a next release is another minor release, or a new major release. There is no predefined timing for next major version. The decision is typically based on semantic versioning considerations, such as whether API changes are needed or older capabilities need to be removed. Other considerations may also apply, for example significant upgrade steps may motivate a shift to a new major version.
+
+If a major release is not delivered for an extended period of time, the maintainers may designate a subsequent minor release as the next LTS release, for example if v3.1 is a LTS release and there is no need to increment to 4.0 for several quarters, the maintainers may decide to designate v3.4 as the next LTS release.
+
+## LTS 3rd digit patch releases
+
+For LTS releases, 3rd digit patch releases will be provided for bug and security fixes approximately every three months (may be more frequent if important fixes become available sooner, and may be deferred if there are no fixes to publish). Significant new features and other changes will not be included in 3rd digit patch releases, in order to ensure the stability of the LTS release and reduce the risk of functional regressions and bugs.
+
+It is worth noting that the v1.4 LTS release did not follow the above principles. Had the v1.4 LTS release followed the above principles, the Raft ordering service feature would likely have been introduced in a v1.5 release, and Kafka to Raft migration in a v1.6 release. We mention only to illustrate the above principles.
+
+## LTS release duration
+
+Once a *subsequent* LTS release is designated, users can expect 3rd digit patch releases to address critical bugs on the *prior* LTS release for approximately 9 more months. The maintainers will determine which fixes should to be backported to the latest LTS release, versus which fixes need to be further backported to the prior LTS release. 3rd digit patch releases should be expected less frequently for prior LTS releases, since only critical fixes will be published. The overlap period is intended to provide users a time window to upgrade their nodes, and any channel capabilities (if required).

```

Comment:
Somehow, 9 month is a little shorter as "long term".
Typically it's several years for most existing software from https://en.wikipedia.org/wiki/Long-term_support.
I think it should make sense to extend it to 12 month at least.

Reply:
9 months is the time for which two LTS releases will live simultaneously. This is saying once 2.x is marked as LTS we will continue delivering bug fixes to 1.4 for 9 months. It doesn't speak to length of time 2.x will actually be LTS for, that could be several years, and then once we deliver a new LTS after 2.x, then 2.x will continue to receive bug fixes for 9 months, while living simultaneously with the LTS after 2.x

Reply:
Make sense!
Besides, it would be better describe the interval between two LTS versions, too.